installation_check.ipynb

```diff
@@ -256,6 +256,48 @@
     "                             color_max)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "grid_size, start_time, end_time = 10, 0, 2\n",
+    "for a in animals:\n",
+    "    locomotion.heatmap.getSurfaceData(a, grid_size, start_time, end_time)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "variables = ['Y', 'Velocity', 'Curvature']\n",
+    "start_time, end_time = 0,1\n",
+    "norm_mode = 'spec'\n",
```

Comment:
Why are variables, start_time, end_time, and norm_mode being defined before calling computeAllCSD()?  They aren't arguments for that routine...

Reply:
I think this was copied over form the BDD version? accidental redundancy

Reply:
I just fixed it.